profile/README.md

```diff
@@ -0,0 +1,6 @@
+# Welcome To The Odin Project!
+
+[The Odin Project (TOP)](https://www.theodinproject.com/) is an open-source curriculum for learning full-stack web development. Our mission is to provide a comprehensive curriculum to learn web development for free. We help users learn the skills and build the impressive portfolio of projects they need to get hired as a web developer.
```

Comment:
I think until we can align on a mission statement, we could just have for now:

```suggestion
[The Odin Project (TOP)](https://www.theodinproject.com/) is an open-source curriculum for learning full-stack web development. We help users learn the skills and build the impressive portfolio of projects they need to get hired as a web developer.
```

Though from there, do we want to limit this verbiage to only "web development", or go for a more general "programming" or rather "software engineer"?

Reply:
Done, this was lifted from the README of the app repo, but happy to take out mission. 
>Though from there, do we want to limit this verbiage to only "web development", or go for a more general "programming" or rather "software engineer"?

I think web developer is fairly descriptive / representative of the current state of the curriculum and the path most learners would take, but I could see some value in making it more general.